gamification/tests/test_models.py

```diff
@@ -341,3 +345,18 @@ def test_add_badges_method(self):
 
         # After applying add_badge method
         self.assertEquals(test_participant.badges.count(), 0)
+
+    def test_update_participants_data_with_meta_review(self):
+        # Test the participant who has completed the meta-review activity
+        Participant.objects.create(username='shikharvaish28')
+        test_meta_review_participant = Participant.objects.get(
+                                       username='shikharvaish28')
```

Comment:
This is wrong. You are mixing up gamification participants and meta-review participants. You also need to import `Participant` from `meta-review` app.